ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -3231,3 +3231,49 @@ def test093_select_large_number_of_test_units_in_one_testplan(self):
                     self.assertIn(testunit, result['test_units'])
 
 
+    def test095_update_department_multiple_contacts(self):
+        """
+        Orders: Child table: Department Approach:  Any update in the order field should be reflected on the order
+        child table In case I have multiple contacts.
+        LIMS-5774
+        """
+        self.info('create 3 contacts with multiple departments')
+        contacts = []
+        contacts_departments = []
+        for i in range(3):
+            response1, payload1 = ContactsAPI().create_contact_with_multiple_departments()
+            self.assertEqual(response1['status'], 1)
+            contacts.append(response1['company']['name'])
+            contacts_departments.append([dep['text'] for dep in payload1['departments']])
+        self.info('create new order with multiple contacts and choose multiple departments related to the '
+                  'different contacts selected')
+        order_no, departments_list = self.order_page.create_multiple_contacts_new_order(contacts=contacts,
```

Comment:
you can update https://github.com/modeso-open-source/1lims-automation/blob/dfa3866d00b9b4600e97e261f7921eef5e57a321/api_testing/apis/orders_api.py#L361
to create order with 3 contacts with departments using API